### PR TITLE
Udpate to non-sharded mosquito index genome

### PR DESCRIPTION
**Pull request by morsecodist**
# Description

This updates the configuration to use the new non-sharded mosquito indexes.

# Tests

I manually added these indexes to staging with a different name and ran several different runs against them. They all worked out.

### CODE REVIEW

File: db/migrate/20191009171351_mosquito_genome_update_2019_10_02.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class MosquitoGenomeUpdate20191002 < ActiveRecord::Migration[5.1]
+  def up
+    # The previous mosquito genome was not added via a migration so we need to destroy it
+    old_hg = HostGenome.find_by(name: "Mosquito").destroy
```
Comment (gregdingle) [inline db/migrate/20191009171351_mosquito_genome_update_2019_10_02.rb]: Won't this break references between all current samples and their host genomes? (foreign key constraints should prevent this) 

Why not create a Mosquito v2?

File: db/migrate/20191009171351_mosquito_genome_update_2019_10_02.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    old_hg.destroy if old_hg
```
Comment (gregdingle) [inline db/migrate/20191009171351_mosquito_genome_update_2019_10_02.rb]: it looks like `.destroy` is called twice

File: db/migrate/20191009171351_mosquito_genome_update_2019_10_02.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+class MosquitoGenomeUpdate20191002 < ActiveRecord::Migration[5.1]
+  def change
```
Comment (gregdingle) [inline db/migrate/20191009171351_mosquito_genome_update_2019_10_02.rb]: is this actually reversible? rails must be really magic then!

if not then change the method to `up`

Comment (morsecodist) [reply db/migrate/20191009171351_mosquito_genome_update_2019_10_02.rb]: Nope, thanks for clearing that up.

**Pull request merged**